components/device-mgt/org.wso2.carbon.device.mgt.ui/src/main/resources/jaggeryapps/devicemgt/api/device-api.jag

```diff
@@ -73,6 +73,31 @@ if (!user) {
         } else {
             result = 400;
         }
+    }else if (uriMatcher.match("/{context}/api/devices/sketch/generate_link")) {
+
+        if (!request.getContent()){
+            log.error("Request Payload Is Empty");
+            // HTTP status code 400 refers to - Bad request.
+            result = 400;
+        }else{
```

Comment:
Here also need a space after the curly brace and the key word 'else'. Please format the code "} else {"

Reply:
This is fixed 0ce103e1ad33617f640d5c03b57477f2b5adb4db